torq.q

```diff
@@ -400,6 +400,7 @@ readprocfile:{[file]
 		// map hostnames in res to order of preference, select most preferred
 		first res iasc prefs?res[`host];
 		first res];
+	if[null output`port;.err.ex[`readprocfile;"Port not found in process.csv. Parameters include host: ", string output`host, ", procname: ", string output`procname, ", proctype: ",string output`proctype;1]]; 
```

Comment:
Just needs some squares to avoid joining strings and symbols (and proctype first just for neatness):
```suggestion
	if[null output`port;.err.ex[`readprocfile;"Port not found in process.csv. Parameters include host: ", string[output`host], ", proctype: ", string[output`proctype], ", procname: ",string output`procname;1]]; 
```